password/impl/src/main/java/org/wildfly/security/password/impl/MaskedPasswordImpl.java

```diff
@@ -210,7 +211,9 @@ public boolean equals(final Object obj) {
             return false;
         }
         MaskedPasswordImpl other = (MaskedPasswordImpl) obj;
-        return iterationCount == other.iterationCount && Arrays.equals(initialKeyMaterial, other.initialKeyMaterial) && Arrays.equals(salt, other.salt) && Arrays.equals(maskedPasswordBytes, other.maskedPasswordBytes) && Arrays.equals(initializationVector, other.initializationVector) && algorithm.equals(other.algorithm);
+        return iterationCount == other.iterationCount && Arrays.equals(initialKeyMaterial, other.initialKeyMaterial)
```

Comment:
Is this "Arrays.equals" intentional?

Reply:
It is. Based on https://github.com/wildfly-security/wildfly-elytron/pull/1779.